addons/mail/static/src/core/persona_service.js

```diff
@@ -55,6 +55,30 @@ export class PersonaService {
             this.store.registeredImStatusPartners?.push(persona.id);
         }
     }
+
+    /**
+     * List of known partner ids with a direct chat, ordered
+     * by most recent interest (1st item being the most recent)
+     *
+     * @returns {[integer]}
```

Comment:
`integer` doesn't exist in javascript, you need to use `number`. In this case it's `@returns {number[]}`

Reply:
integer comes into the type from below and that's why I make it consistent. 
```
    /** @type {integer} */
    chatPartnerId;
```
And there are still integers in the type definition. Maybe we will correct them into number but in another big PR.